### DISCUSSION TITLE
Discord notifications fail

### DISCUSSION THREAD

**Discussion by jfarris10** [sct/overseerr#1038] [Help]
I can't seem to get Discord notifications working. See errors below:

`[debug][Notifications]: Sending discord notification`
`[error][Notifications]: Request failed with status code 400 {"response":{"embeds":["0"]}}`

I'm able to configure a generic Webhook notification agent in Overseerr and it works fine (including relaying that notification to a Discord webhook). 

Anyone facing a similar issue?

Useful info:

* Overseer version 1.20.0 (the issue has occurred since ~1.16.0)
* Deployed in Docker

**Comment by dragonnest21** [comment]
In my case its the opposite. I can't seem to get generic webhook to work with discord yet its easy to make it work with discord notification. Mind showing me a screenshot how you set up yours? Would be really helpful.

**Comment by samwiseg0** [comment]
`{"response":{"embeds":["0"]}}` Means discord rejected the message because it is not formatted correctly. Verify you are sending the data correctly. Also no embed can have `null` data in any fields. I would check that first.
